capstone-overview/logistics.md

```diff
@@ -0,0 +1,12 @@
+## Capstone Logistics to Review
```

Comment:
This info looks good! Is this doc meant to be the student facing? If so, I might make a few changes to the layout and make the voice consistent.